packages/testsuite/cypress/e2e/mail/test-configuration-subsystem-mail-attributes.cy.ts

```diff
@@ -0,0 +1,102 @@
+describe("TESTS: Configuration => Subsystems => Mail", () => {
+  const address = ["subsystem", "mail", "mail-session", "my-mail-session"];
+  const configurationFormId = "mail-session-form";
+  const formColumNameName = "name";
+  const formColumValueName = "my-mail-session";
+  const formColumNameDebug = "debug";
+  const formColumNameFrom = "from";
+  const formColumValueFrom = "newFromValue";
+  const formColumValueDefaultFrom = "DefaultFromValue";
+  const formColumNameJNDI = "jndi-name";
+  const formColumValueJNDI = "java:jboss/mail/Custom";
+  const formColumValueDefaultJNDI = "java:jboss/mail/MyDefault";
+
+  let managementEndpoint: string;
+
+  before(() => {
+    cy.startWildflyContainer().then((result) => {
+      managementEndpoint = result as string;
+    });
+  });
+
+  after(() => {
+    cy.task("stop:containers");
+  });
+
+  beforeEach(() => {
+    cy.navigateTo(
+      managementEndpoint,
+      "mail-session;name=" + formColumValueName
+    );
+  });
+
+  it("Add custom mail session", () => {
+    cy.navigateTo(
+      managementEndpoint,
+      "configuration;path=configuration~subsystems!css~mail"
+    );
+    cy.get("#mail-session-add").click();
+    cy.text(configurationFormId, formColumNameName, formColumValueName);
+    cy.text(configurationFormId, formColumNameFrom, formColumValueDefaultFrom);
+    cy.text(configurationFormId, formColumNameJNDI, formColumValueDefaultJNDI);
+    cy.get(
+      'div.modal-footer > button.btn.btn-hal.btn-primary:contains("Add")'
+    ).click();
+    cy.verifySuccess();
+    cy.verifyAttribute(
+      managementEndpoint,
+      address,
+      formColumNameFrom,
+      formColumValueDefaultFrom
+    );
+    cy.verifyAttribute(
+      managementEndpoint,
+      address,
+      formColumNameJNDI,
+      formColumValueDefaultJNDI
+    );
+  });
+
+  it("Toggle debug", () => {
+    cy.getDefaultValue(managementEndpoint, address, formColumNameDebug).then(
+      (defaultValue: boolean) => {
+        cy.editForm(configurationFormId);
+        cy.flip(configurationFormId, formColumNameDebug, defaultValue);
+        cy.saveForm(configurationFormId);
+        cy.verifySuccess();
+        cy.verifyAttribute(
+          managementEndpoint,
+          address,
+          formColumNameDebug,
+          !defaultValue
+        );
+      }
+    );
+  });
+
+  it("Edit from", () => {
+    cy.editForm(configurationFormId);
+    cy.text(configurationFormId, formColumNameFrom, formColumValueFrom);
+    cy.saveForm(configurationFormId);
+    cy.verifySuccess();
+    cy.verifyAttribute(
+      managementEndpoint,
+      address,
+      formColumNameFrom,
+      formColumValueFrom
+    );
+  });
+
+  it("Edit JNDI name", () => {
```

Comment:
please keep the attribute name in lower case separated by a hyphen, e.g `jndi-name`